ting_openformat.field.inc

```diff
@@ -522,6 +529,68 @@ function _ting_openformat_parse_analytic_content_element($element) {
 
 }
 
+/**
+ * Helper function to parse tracks
+ * @param $tracks
+ * @param null $entity
+ * @param array $options
+ * @return array|null
+ */
+function _ting_openformat_parse_tracks($tracks, $entity = NULL, $options = array()) {
+  if (!isset($tracks) || !isset($tracks[1]['track'])) {
+    return null;
+  }
+  $render_array = array();
+  foreach ($tracks as $element) {
+    $render_array = _ting_openformat_parse_tracks_element($element['track'][0]);
+  }
+
+  $render_array = ting_openformat_text_summary($render_array);
+  return $render_array;
+}
+
+/**
+ * Helper function to parse track elements
+ * @param $element
+ * @return string
+ */
+function _ting_openformat_parse_tracks_element($element) {
+  $render_array = array();
+  if ($element === NULL) {
+    return $render_array;
+  }
+  $count = count($element);
+  $i = 0;
+  foreach ($element as $key => $value) {
+    $i++;
+    end($value);
+    $last_key = key($value);
+    foreach ($value as $k => $v) {
+      $markup = _ting_openformat_get_markup_for_field($v, " ");
```

Comment:
Er $markup altid et array eller kan det f.eks. være null eller lign. du skal tage højde for?

Reply:
godt spørgsmål.... undersøger...